spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Ingredients index page' do
+    it "displays a list of all the ingredients with name and cost" do
+        ingredient_1 = Ingredient.create(name: 'Eggs', cost: 2)
+        ingredient_2 = Ingredient.create(name: 'Butter', cost: 4)
+        ingredient_3 = Ingredient.create(name: 'Chocolate Chips', cost: 5)
+
+        visit "/ingredients"
+
```

Comment:
This looks great. You can make it even more robust by using `within` blocks to test.